src/services/Subscriptions/SubscriptionQueries.ts

```diff
@@ -0,0 +1,27 @@
+import { useState } from 'react';
+import { Subscriptions, getSubscriptions } from './SubscriptionApi';
+import useErrorNotification from 'Hooks/useErrorNotification';
+
+export const useFetchSubscriptionsQuery = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const errorNotifier = useErrorNotification();
+
+  const fetchSubscriptions = async (): Promise<Subscriptions | null> => {
```

Comment:
Use react query here.
Although it isn't really needed for caching, it is used at the time level for monitoring of API calls for QE test reference. 
You can see how that works [here](https://github.com/content-services/content-sources-frontend/blob/main/src/App.tsx#L64).

Reply:
sure, good point 👍🏼 I just made it similar to `useFetchFeaturesQuery` which I guess is outdated 😅 resolved [here](https://github.com/content-services/content-sources-frontend/pull/328/commits/92ada247ce4e2cfe19fb4e6adac241f9ae08e54d) 🫡